lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -34,6 +34,11 @@ def as_json
 
         private
 
+        def ensure_valid!(json)
```

Comment:
I'm not following. If the GET returns null in the example you gave, there would be no actual relationship persisted in the system, right? Wouldn't you expect it to update your objects according to the result of the GET you'd just performed?